### DISCUSSION TITLE
天气模块无法使用

### DISCUSSION THREAD

**Discussion by Laeoo** [NSRingo/.github#9] [Q&A 问答] [answered]
在ipad os18.1中，启用模块后天气仍显示cn aqi
设备为ipadmini7国行蜂窝版本
qx日志显示
2024/11/03 03:28:36:585  <error>      [QTXDataParser] Left side MitM error for api.smoot.apple.cn with optional ssl code -9808.

**Comment by VirgilClyne** [comment, marked as answer]
`api.smoot.apple.cn` 是 `Siri 搜索与建议` 使用的域名，与天气无关
